DEVELOPER.md

```diff
@@ -14,10 +14,12 @@ See http://kafka.apache.org/documentation.html#consumerconfigs for details about
 
     input {
         kafka {
-            topic_id => ... # string (required), The topic to consume messages from
+            topic_id => ... # string (required), The topic to consume messages from. Can be a java regular expression for whitelist of topics.
```

Comment:
That's fine with me.
On Feb 24, 2015 5:46 PM, "Tal Levy" <notifications@github.com> wrote:

> In DEVELOPER.md
> <https://github.com/logstash-plugins/logstash-input-kafka/pull/25#discussion_r25301370>
> :
>
> > @@ -14,10 +14,12 @@ See http://kafka.apache.org/documentation.html#consumerconfigs for details about
> >
> >      input {
> >          kafka {
> > -            topic_id => ... # string (required), The topic to consume messages from
> > +            topic_id => ... # string (required), The topic to consume messages from. Can be a java regular expression for whitelist of topics.
>
> why not have a similar behavior to the jruby-kafka parameters?
>
>
> https://github.com/joekiller/jruby-kafka/blob/master/lib/jruby-kafka/group.rb#L181-L206
>
> We can split them out and make them mutually exclusive?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logstash-plugins/logstash-input-kafka/pull/25/files#r25301370>
> .
>